tests/integration/cartography/intel/github/test_repos.py

```diff
@@ -401,25 +401,25 @@ def test_sync_github_dependencies_end_to_end(neo4j_session):
     # _ensure_local_neo4j_has_test_data has already called sync, now we test that the sync worked. Mock GitHub API data should
     # be transofrmed and in the Neo4j database.
 
-    # Create expected IDs with simple format: canonical_name|version
+    # Create expected IDs with format: canonical_name|requirements
     repo_url = "https://github.com/cartography-cncf/cartography"
     react_id = "react|18.2.0"
-    lodash_id = "lodash"
-    django_id = "django|4.2.0"
+    lodash_id = "lodash|^4.17.21"
+    django_id = "django|==4.2.0"
     spring_core_id = "org.springframework:spring-core|5.3.21"
 
     # Assert - Test that new GitHub dependency graph nodes were created
     # Note: Database also contains legacy Python dependencies, so we check subset
     expected_github_dependency_nodes = {
         (react_id, "react", "18.2.0", "npm"),
-        (lodash_id, "lodash", None, "npm"),
-        (django_id, "django", "4.2.0", "pip"),
+        (lodash_id, "lodash", "^4.17.21", "npm"),
+        (django_id, "django", "==4.2.0", "pip"),
```

Comment:
For pip deps doesn't github return like "= 4.2.0"? Let's make sure the test data accurately represents what is returned from the API